docusaurus/docs/get-started/set-up-development-environment.mdx

```diff
@@ -130,3 +141,252 @@ You will also notice that the `docker-compose.yaml` file also has the following
 ```
 
 they are necessary to allow delve to attach to the running process and debug it and should not be used in production environments.
+
+## Set up GitHub workflows
+
+Automate your development process to minimize errors and make it faster and more cost-efficient. The `create-plugin` tool helps you to configure your GitHub actions workflows to help automate your development process.
+
+### The CI workflow
+
+The CI (`ci.yml`) workflow is designed to lint, type check, and build the frontend and backend. It is also used to run tests on your plugin every time you push changes to your repository. The `create-plugin` tool helps to catch any issues early in the development process, before they become bigger problems.
```

Comment:
It also runs the playwright-tests (`grafana/plugin-actions/e2e-version@main`) which creates overlap with this page: https://grafana-dev.com/developers/plugin-tools/e2e-test-a-plugin/ci.

Reply:
the text does still say:
> It is also used to run tests

Would this address things for now and we can have more of a discussion about how to better combine the topics if required?

```suggestion
The CI (`ci.yml`) workflow is designed to lint, type check, and build the frontend and backend. It is also used to run tests on your plugin every time you push changes to your repository. The `create-plugin` tool helps to catch any issues early in the development process, before they become bigger problems. For more information on E2E testing as part of the CI workflow, refer to our [documentation](../e2e-test-a-plugin/ci.md).
```

Reply:
I think so. Added this fix in this commit: https://github.com/grafana/plugin-tools/pull/993/commits/84351e9ca091635e521ab4f4f098d354951fa9a8